src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,60 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        Book testBook1 = bookDao.create(new Book("Harry Potter and the Philosophers Stone",
+                BigDecimal.valueOf(9.99)));
+
+        //findById method test
+        Optional<Book> testFindBook = bookDao.findById(1L);
+        if (testFindBook.isPresent()) {
+            Book test1 = testFindBook.get();
+            System.out.println("Find by ID1 book: " + System.lineSeparator() + test1);
+        }
+
+        //Update method test
+        testBook1.setPrice(BigDecimal.valueOf(5.99));
+        bookDao.update(testBook1);
+        System.out.println(System.lineSeparator()
+                + "Suggested price changed from 9.99 to 5.99 on book with ID 1");
+        bookDao.findAll().forEach(System.out::println);
+
+        Book testBook2 = bookDao.create(new Book("Harry Potter and the Chamber of Secrets",
```

Comment:
As per the previous comment, ensure that the `books.title` column is not set to UNIQUE in the database schema to avoid issues when creating multiple books with the same title.